src/thunderbots_msgs/msg/Primitive.msg

```diff
@@ -0,0 +1,18 @@
+# This message generically defines a low-level Primitive
+
+# The name of the primitive. We use a string because it is
+# easier to read and debug than an integer.
+string primitive_name
+
+# The id of the robot the Primitive corresponds to
+uint32 robot_id
+
+# All of our Primitive parameters can be represented and passed
+# as a double. For example, a destination x-coordinate, orientation.
+# or kicking speed.
+float64[] parameters
```

Comment:
Yea it would be, but we tend to use `double` by default when dealing with floating point in our code, so having this as float64 (equivalent to double) saves us some casting. I don't think we would see a significant increase in speed to make up for it.